modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -182,6 +182,10 @@ inline IE::Blob::Ptr wrapIE(const cv::MediaFrame::View& view,
             auto uv_plane = cv::Mat(desc.size / 2, CV_8UC2, view.ptr[1], view.stride[1]);
             return cv::gapi::ie::util::to_ie(y_plane, uv_plane);
         }
+        case cv::MediaFormat::GRAY: {
+            auto gray = cv::Mat(desc.size, CV_8UC1, view.ptr[0], view.stride[0]);
+            return wrapIE(gray, cv::gapi::ie::TraitAs::IMAGE);
```

Comment:
Will view still leave? I mean if we let `cv::Mat` created on top of `View` memory to go further, how is `View` lifetime will be handled and destroyed until `cv::Mat` is used?

Reply:
No idea. Just copy-pasted it from line 177 operating with BGR.